openpower/package/ppe42-gcc/ppe42-gcc.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-PPE42_GCC_VERSION ?= d8a1bac8634033a3edd4e9a22455f97318718f43
+PPE42_GCC_VERSION ?= f8c463f16d47773cfd9946a8948a50efde3fee14
```

Comment:
can you please include a "git shortlog d8a1bac8634033a3edd4e9a22455f97318718f43..f8c463f16d47773cfd9946a8948a50efde3fee14" in the commit message so we can easily spot from git history of op-build what has changed?